src/main/java/uk/gov/companieshouse/pscstatementdataapi/config/SimpleAuthenticationInterceptor.java

```diff
@@ -0,0 +1,45 @@
+package uk.gov.companieshouse.pscstatementdataapi.config;
+
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.stereotype.Component;
+import org.springframework.web.filter.OncePerRequestFilter;
+
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+@Component
+public class SimpleAuthenticationInterceptor extends OncePerRequestFilter {
```

Comment:
Could this name be more descriptive if changed to something that references that it is a filter which filters Eric requests?

EricAuthFilter??